src/scripts/templates/active/Active default template.js

```diff
@@ -4,7 +4,14 @@
 // Note that new active scripts will initially be disabled
 // Right click the script in the Scripts tree and select "enable"  
 
-
+/**
+ * Scans a "node", i.e. an individual entry in the Site Tree.
+ * 
+ * @param as the ActiveScan parent object that will do all the core interface tasks 
```

Comment:
Hmm ok I'll try to revise to be more specific, trying to hit a balance between dev comments and user comments :smiley: